components/producers/ossf-scorecard/main.go

```diff
@@ -36,19 +37,22 @@ func main() {
 }
 
 func parseIssues(out *ScorecardOut) []*v1.Issue {
+	slog.Info("read ", slog.Int("numChecks", len(out.Checks)))
 	issues := []*v1.Issue{}
 	repo := out.Repo.Name
 	commit := out.Repo.Commit
 	for _, r := range out.Checks {
-		desc, _ := json.Marshal(r)
+		var desc string
```

Comment:
Why did we change this from json?

Reply:
because it's going into a generic string, dumping json into a string field is usually ill-advised